### PR TITLE
Log NFT-ID's in user's account

### PR DESCRIPTION
**Pull request by ratik21**

### CODE REVIEW

File: examples/nft/scripts/transfer/common.js [hunk 1]
```diff
@@ -26,11 +27,18 @@ exports.printLocalNFT = async function (deployer, account, appId) {
     try {
         const localState = await readLocalStateSSC(deployer, account, appId);
         // each nft is stored as a one record in user store
-        let count = 0;
-        if (localState !== undefined)
-            count = localState.length
-        console.log('NFT balance of', account, ':', count);
+        let holdings = "";
+        if (localState === undefined) {
+            holdings = "null";
```
Comment (robert-zaremba) [inline examples/nft/scripts/transfer/common.js:32]:
```suggestion
            holdings = "none";
```

File: examples/nft/scripts/transfer/common.js [hunk 2]
```diff
@@ -26,11 +27,18 @@ exports.printLocalNFT = async function (deployer, account, appId) {
...
+        }
+        else {
+            for(const l of localState) {
+                const key = decode(Buffer.from(l.key, 'base64'));
+                holdings += key.toString() + ' ';
+            }
+        }
+        console.log('NFT holdings of', account, ':', holdings);
```
Comment (robert-zaremba) [inline examples/nft/scripts/transfer/common.js:40]: can you show an example output please?

Comment (ratik21) [reply examples/nft/scripts/transfer/common.js:40]: Sure.
- Before Transfer
image
- After Transfer
image

Comment (robert-zaremba) [reply examples/nft/scripts/transfer/common.js:40]: thanks. I will update the message text.

File: examples/nft/scripts/transfer/common.js [hunk 3]
```diff
@@ -26,10 +27,17 @@ exports.printLocalNFT = async function (deployer, account, appId) {
...
+        if (localState === undefined) {
+            holdings = "none";
+        }
+        else {
+            for(const l of localState) {
+                const key = decode(Buffer.from(l.key, 'base64'));
+                holdings += key.toString() + ' ';
+            }
+        }
+        console.log('NFT holdings of', account, ':', holdings);
```
Comment (robert-zaremba) [inline examples/nft/scripts/transfer/common.js:40]:
```suggestion
        console.log('%s account holds app(%s) NFTs: ', account, appId, holdings);
```

File: examples/nft/scripts/transfer/common.js [hunk 4]
```diff
@@ -26,10 +27,17 @@ exports.printLocalNFT = async function (deployer, account, appId) {
...
-            count = localState.length
-        console.log('NFT balance of', account, ':', count);
+        let holdings = "";
+        if (localState === undefined) {
+            holdings = "none";
+        }
+        else {
+            for(const l of localState) {
+                const key = decode(Buffer.from(l.key, 'base64'));
+                holdings += key.toString() + ' ';
```
Comment (robert-zaremba) [inline examples/nft/scripts/transfer/common.js:37]: `holdings` should be an array. Avoid using string concatenation in a loop. Instead use `array.push` and after the loop you can use `array.join` to create a string.

**Review by robert-zaremba — APPROVED** [review]
pre-approving.

File: examples/nft/scripts/transfer/common.js (hunk 4 above)
Comment (ratik21) [reply examples/nft/scripts/transfer/common.js:37]: ok :+1:

**Pull request merged**